src/passwordless/email_sent_confirmation.jsx

```diff
@@ -13,7 +13,7 @@ class ResendLink extends React.Component {
   render() {
     const { label, onClick } = this.props;
     return(
-      <a className="auth0-lock-resend-link" href="" onClick={onClick}>
+      <a className="auth0-lock-resend-link" href="#" onClick={onClick}>
```

Comment:
I've chosen the shortest path.

React has its own event system so to really prevent the default or stop the propagation I would need to obtain the native event, which is more trouble that just adding the hash (`event.preventDefault()` is being [called](https://github.com/auth0/lock-passwordless/blob/a251ce07efd34ad80a5266a63e5f91fb94c307ae/src/passwordless/email_sent_confirmation.jsx#L51) to avoid the navigation). 

Changing the `<a>` tag for a `<button>` requires styling changes.